.octopus/deployment_process.ocl

```diff
@@ -0,0 +1,89 @@
+step "deploy-to-internal-teamcity" {
+    name = "Deploy to Internal TeamCity"
+
+    action {
+        action_type = "Octopus.Script"
+        environments = ["test"]
+        notes = "We can't do this yet because we can't upgrade a plugin without restarting TeamCity server. We hope this feature will be enabled soon. See [TW-12604](https://youtrack.jetbrains.com/issue/TW-12604)."
+        properties = {
+            Octopus.Action.Script.ScriptBody = "echo \"Not yet implemented\""
+            Octopus.Action.Script.ScriptSource = "Inline"
+            Octopus.Action.Script.Syntax = "Bash"
+        }
+        worker_pool = "hosted-ubuntu"
+    }
+}
```

Comment:
We should consider just removing this step.

Reply:
I have removed this, as it is unused for now